docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -1,23 +1,27 @@
 [role="xpack"]
 [testenv="basic"]
 [[set-up-lifecycle-policy]]
-== Configure lifecycle policy [[ilm-policy-definition]]
+== Configure a lifecycle policy [[ilm-policy-definition]]
 
 For {ilm-init} to manage an index, a valid policy 
 must be specified in the `index.lifecycle.name` index setting. 
 
-To configure a lifecycle policy for rolling indices, 
-you create the policy and add it to the index template.
+To configure a lifecycle policy for <<index-rollover, rolling indices>>, 
+you create the policy and add it to the <<indices-templates, index template>>.
 
 To use a policy to manage an index that doesn't roll over,
-you can specify the policy directly when you create it.
+you can specify a lifecycle policy when you create it.
+
+{ilm-init} policies are stored in the global cluster state and can be included in snapshots
+by setting `include_global_state` to `true` when you <<snapshots-take-snapshot, take the snapshot>>. 
+Restoring {ilm-init} policies from a snapshot is all-or-nothing. 
+The entire global state, including all policies, is overwritten when you restore a snapshot.
```

Comment:
This is a bit misleading to me, because it's only policies that have the same names that are overwritten, policies that exist locally but not in the snapshot are not removed. Maybe we can clarify it a little bit?